theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -7,19 +7,44 @@ set -xe
 # * the web node must have the freight class applied
 
 # Find the deps in the build-dir from the previous step
-DEB_PATH=./debian/${os}/build-${project}
+DEB_PATH=$(find ./ -name build-${project})
 
 # Upload all builds to stagingdeb for testing
-echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
-export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
-USER=freightstage
```

Comment:
I'd keep the two lines here, and only specialcase the "freight" case when uploading plugin builds.

Reply:
See if my change matches what you were aiming for.

Reply:
Yes, absolutely!